### ISSUE TITLE
Last summary-infobox during user-tutorial is out of border (in Firefox)

### ISSUE THREAD

**Issue by maw42** [openstreetmap/iD#10634]
### URL

https://www.openstreetmap.org/edit#map=19/49.040941/12.122252

### How to reproduce the issue?

Click through the OpenStreetmap-tour which is part of the tutorial session right after registrating an account.
Then at the end, the last infobox is partially out of the browser-window and not seen. Browser is  LibreWolf 133.0.3-1 (a fork of Firefox), scaling in Browser is 100%, in operating system (archlinux with KDE / Plasma GUI) the UI-scaling is 150%, but should be irrelevant.
When reducing scaling in the browser, the window is shown completely again and in the bounds of the browser again.

Probably not everyone will know that changing browser-scaling is necessary, and will thus be confused.

### Screenshot(s) or anything else?

![finish_scaling_wrong](https://github.com/user-attachments/assets/9de9e4b0-9391-403c-9659-1a03e30a8555)

Here is a paste of the browser-element, picked via the webinspector:

```
<div class="curtain-tooltip popover tooltip arrowed in top" style="top: 80.1833px; left: 1163.04px;">
    <div class="popover-arrow"></div>
    <div class="popover-inner" style="top: 0px;"><span class="localized-text" lang="en-US">You're now ready to edit
            OpenStreetMap!<br><br>You can replay this walkthrough anytime or view more documentation by pressing the
            <svg class="icon inline">
                <use xlink:href="#iD-icon-help"></use>
            </svg> <span class="localized-text" lang="en-US">Help</span> button or the <kbd><span class="localized-text"
                    lang="en-US">H</span></kbd> key.</span>
        <div class="button-section"><button href="#" class="button action"><span class="localized-text"
                    lang="en-US">OK</span></button></div>
    </div>
</div>

```
and a screenshot of the CSS-boxes:
![finish_scaling_wrong_2](https://github.com/user-attachments/assets/eb96877b-5648-4c93-bbd1-7fd600748f0a)

### Which browsers are you seeing this problem on?

Firefox

**Comment by AntonKhorev** [comment]
Isn't it the same as #10233 and #10267?

**Comment by maw42** [comment]
yes, seems to be same issue, just occuring at different steps. #10233 and #10267 are missing some technical details, maybe this issue is helpful in debugging / fixing. Feel free to link to the other issues and mark as duplicate as needed. Thank you :)

**Comment by AntonKhorev** [comment]
> just occuring at different steps

Is there any step when it doesn't occur? I think I've seen it during various steps. But now I opened the tutorial on my local machine and it works fine.

Were shaded/unshaded areas moving around during the tutorial when infoboxes were misplaced? I think I couldn't see them when bugs were happening.

**Comment by maw42** [comment]
I did not see shaded areas moving around, but noticed that I was using a lower and different system resolution (in Display settings in KDE Plasma) than recommended, so had a black area at the top and bottom of my screen (so I used 1920x1080 , 16:9, or some similar resolution). When I switched to 2256x1504, ratio 3:2), it did no longer occur. But in both cases the LibreWolf-browser was maximized, so should not be a factor...
Maybe resolution was initially too low, or somehow the system-resolution is processed instead of the browser-solution somewhere?

**Comment by tyrasd** [comment]
To reproduce: it is apparently relevant that the browser window is narrow in the vertical direction.
